qa/tasks/ceph_manager.py

```diff
@@ -1551,11 +1551,10 @@ def do_rados(self, remote, cmd, check_status=True):
         """
         Execute a remote rados command.
         """
-        testdir = teuthology.get_testdir(self.ctx)
         pre = [
             'adjust-ulimits',
             'ceph-coverage',
-            '{tdir}/archive/coverage'.format(tdir=testdir),
+            f'{self.testdir}/archive/coverage'
```

Comment:
might want to add a comma at the end of this line.  i have
```
2021-03-13T06:29:38.588 DEBUG:teuthology.orchestra.run.smithi152:> sudo adjust-ulimits ceph-coverage /home/ubuntu/cephtest/archive/coveragetimeout 0 ceph --cluster ceph --admin-daemon /var/run/ceph/ceph-mon.a.asok config show
2021-03-13T06:29:38.613 INFO:teuthology.orchestra.run.smithi152.stderr:/bin/ceph-coverage: line 25: exec: 0: not found
2021-03-13T06:29:38.614 DEBUG:teuthology.orchestra.run:got remote process result: 127
```

when testing this change. see https://pulpito.ceph.com/kchai-2021-03-13_05:46:44-rados-wip-kefu-testing-2021-03-13-1213-distro-basic-smithi/5960455/

for more instances, see https://pulpito.ceph.com/kchai-2021-03-13_05:46:44-rados-wip-kefu-testing-2021-03-13-1213-distro-basic-smithi/

Reply:
Fixed.